Server/mods/deathmatch/logic/CStaticFunctionDefinitions.h

```diff
@@ -700,6 +700,7 @@ class CStaticFunctionDefinitions
     // Account set funcs
     static CAccount* AddAccount(const SString& strName, const SString& strPassword, bool bAllowCaseVariations, SString& strOutError);
     static bool      RemoveAccount(CAccount* pAccount);
+    static bool      SetAccountSerial(SString strName, const SString strSerial);
```

Comment:
SString is no longer used, please use std::string instead

Reply:
adjusted 62ebf06